src/osx/cocoa/window.mm

```diff
@@ -3933,9 +3955,18 @@ new wxCocoaGesturesImpl(this, m_osxView, eventsMask)
     // this window.
     (void)SetupCursor(event);
 
-    wxMouseEvent wxevent(wxEVT_LEFT_DOWN);
-    SetupMouseEvent(wxevent , event) ;
-    return GetWXPeer()->HandleWindowEvent(wxevent);
+    bool processed = false;
+    for ( auto& wxevent : TranslateMouseEvent(event) )
+    {
+        // Even if this event was processed, still continue with the other
+        // eveents, if any.
```

Comment:
Typo.
```suggestion
        // events, if any.
```

Reply:
Thanks, I'll fix this (and the other) typo(s) when I merge it, if it works.